src/index.js

```diff
@@ -198,12 +198,12 @@ function setupParadigmSizeToggleButton() {
         }
         window.history.replaceState({}, document.title, updateQueryParam('paradigm-size', nextParadigmSize))
         const oldParadigmNode = document.getElementById('paradigm')
-        oldParadigmNode.parentElement.replaceChild(paradigmFrag, oldParadigmNode)
+        oldParadigmNode.firstElementChild.replaceWith(paradigmFrag)
```

Comment:
This may be less brittle going forward if the relevant node is specified using a name, such as a new CSS selector

    oldParadigmNode.querySelector('.paradigmTableOrSomething').replaceWith